source/NVDAObjects/IAccessible/adobeAcrobat.py

```diff
@@ -84,7 +84,7 @@ def initOverlayClass(self):
 	def _get_role(self):
 		try:
 			return normalizeStdName(self.pdDomNode.GetStdName())[0]
-		except (AttributeError, LookupError, COMError):
+		except (AttributeError, LookupError, COMError, TypeError):

```

Comment:
This means that it will hide any other TypeErrors. I think this should instead log at Debug / Debug warning.